pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -80,10 +79,7 @@ func (q *QueryReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (ctrl.
 	}
 
 	// setup dispatcher for vclusterops API
-	dispatcher, err := q.makeDispatcher(q.Log, q.Vdb, nil /*password*/)
-	if err != nil {
-		return ctrl.Result{}, err
-	}
+	dispatcher := q.makeDispatcher(q.Log, q.Vdb, nil /*password*/)
```

Comment:
We still want to return an error here. We should never see the error, but its good defensive programming to handle the case where admintools deployments is set in here.